### ISSUE TITLE
Incompatibility problem with Typo3 10.4.2 since version 2.0.0

### ISSUE THREAD

**Issue by ninar92** [mindshape-GmbH/mindshape_cookie_consent#25]
I'm using Typo3 10.4.22 and PHP 7.4. Since the version 2.0.0 the statistics in the backend dont work anymore. The following error is showing: 

```
Too few arguments to function TYPO3\CMS\Extbase\Persistence\Generic\Mapper\DataMapper::__construct(), 0 passed in .../typo3_src-10.4.22/typo3/sysext/core/Classes/Utility/GeneralUtility.php on line 3480 and at least 7 expected
....
at TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('TYPO3\\CMS\\Extbase\\Persistence\\Generic\\Mapper\\DataMapper')
in .../typo3conf/ext/mindshape_cookie_consent/Classes/Domain/Repository/AbstractStatisticRepository.php line 82
```

I've tested it on more than one site, and the error comes on any site, where i'm using Typo3 10.4.22.

I think there is an version conflict. Can anybody help? 
Thank you!

**Comment by featdd** [comment]
Hi @ninar92,

thanks for the hint, I solved it in 8c93afd6864493be54c693a5d309921b73e826ca.

Will prepare an update as soon as possible.

Greetings
Daniel

**Comment by featdd** [comment]
Hi @ninar92,

sorry for the delay, the update 2.1.1 is now online.

Greetings
Daniel
